server/channels/app/file.go

```diff
@@ -1063,7 +1063,10 @@ func (a *App) DoUploadFileExpectModification(c request.CTX, now time.Time, rawTe
 		}
 	}
 
-	if *a.Config().FileSettings.ExtractContent {
+	// The extra boolean extractContent is used to turn off extraction
+	// during the import process. It is unnecessary overhead during the import,
+	// and something we can do without.
+	if *a.Config().FileSettings.ExtractContent && extractContent {
```

Comment:
Not sure, is it unnecessary during import and necessary when uploading files or always unnecessary? To me the question is how important it is for our customers as it's their data that's getting exported/imported after all. And given this functionality can be controlled through config already it seems rather easy to turn on/off at will. 

I wonder if we could avoid having one more hard-coded boolean and simply control the config knob. And maybe we could expose a flag to `mmctl` to control a temporary override but if it's too much the config alone seems like the cleaner approach. Of course we'd need to document that this process could take longer and perform poorly. 

Speaking of which, I'd also want to use this opportunity to take a step back and consider this feature with all its problems (performance, security, lack of upstream maintenance) and think about if having it in is actually worth it. Maybe there are better alternatives to consider these days? Or maybe deprecation is a real possibility. 

@nab-77 could you offer some perspective and advice perhaps?

Reply:
> And maybe we could expose a flag to mmctl to control a temporary override

Yep, was thinking exactly the same. Add a new flag so that existing behavior is preserved, but allows customers and users to bypass it.

And yes, I'd also urge to reconsider this feature to be deprecated/dropped. Primarily because we don't own the meat of the code. And the repo we depend on has performance/security issues and the owners aren't responsive.